src/registrar/utility/csv_export.py

```diff
@@ -415,7 +415,8 @@ def get_model_annotation_dict(cls, request=None, **kwargs):
             .values(*shared_columns)
         )
 
-        return convert_queryset_to_dict(permissions.union(invitations), is_model=False)
+        members = permissions.union(invitations).order_by("email_display")
```

Comment:
(non-blocking) mIght be a good idea to include a comment here explaining the necessity of the order_by clause (and the bug it fixes) so a future engineer doesn't strip it out for performance or other reasons.